packages/dashboard-e2e/wdio.conf.js

```diff
@@ -22,7 +22,8 @@ function browserstackOptions(opts) {
   return {
     projectName: 'rmf-web',
     buildName: `dashboard-e2e:${process.env.BROWSERSTACK_BUILD || 'local'}`,
-    resolution: '1920x1080',
+    // resolution: '1920x1080',
```

Comment:
Perhaps it would be good to explain in a comment why we decided to comment on this line

Reply:
that's a good idea! I've left in a comment